src/components/AuthForm/typeForms/LinkExternalAuth/LinkExternalAuth.jsx

```diff
@@ -29,7 +29,7 @@ const LinkExternalAuth = ({ href, linkText, icon, className }) => {
 LinkExternalAuth.propTypes = {
   href: PropTypes.string.isRequired,
   linkText: PropTypes.string.isRequired,
-  icon: PropTypes.string /* !TODO: That's not the string ... */,
+  icon: PropTypes.object /* !TODO: That's not the string ... */,
```

Comment:
well... now it is string... not an object. But you must import an SVG as ReactComponent with "vite-plugin-svgr" plugin! And after it will be "element" as I said.

Reply:
+ deleted this props
and use  <IconGoogle />